docs/setting-up/client/postgresql.md

```diff
@@ -72,9 +72,7 @@ Decide which database extension to use, and configure your database server for i
 
 1. [`pg_stat_statements`](#pg_stat_statements), the original extension created by PostgreSQL, part of the `postgresql-contrib` package available on Linux.
 
-2. [`pg_stat_monitor`](#pg_stat_monitor) is a new extension created by Percona. It is based on and compatible with `pg_stat_statements`. `pg_stat_monitor` has all the features of `pg_stat_statements`, but adds *bucket-based data aggregation*.
-
-We recommend choosing only one of these. **If you use both, you will get duplicate metrics.**
+2. [`pg_stat_monitor`](#pg_stat_monitor) is a new extension created by Percona. `pg_stat_monitor` has all the features of `pg_stat_statements` but adds *bucket-based data aggregation*, provides more accurate data, and can expose Query Examples.
 
```

Comment:
Query Examples - should be query examples.

Reply:
Few recommendations:

Apart from having all the features of pg_stat_statements, pg_stat_monitor also adds *bucket-based data aggregation*, provides more accurate data, and can expose query examples.

pg_stat_monitor adds *bucket-based data aggregation*, provides more accurate data, and can expose query examples apart from having all the features of pg_stat_statements.